canvas/fizz-27-py/src/lib/canvasRequest.py

```diff
@@ -0,0 +1,10 @@
+import requests
+import os
+
+def get(url):
+    base_url = os.getenv("CANVAS_BASE_URL")
+    auth_token = os.getenv("ACCESS_TOKEN")
+    r = requests.get(
+        base_url + "/" + url,
+        headers={ "Authorization" : f"Bearer {auth_token}" },)
```

Comment:
I have read several articles on strings in Python and not come across string interpolation with `f` before now! And yet it was added in Python 3.6, I now see, several years ago. Sweet. In all of the articles I had come across before they were still recommending `"Bearer {}".format(auth_token)`, which is an improvement over the older % formatting. I'm going to start using this `f` formatting style.